priv/react_app/src/Components/Forms/StudentSessionForm.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { map } from 'lodash/fp';
+import { Field, reduxForm } from 'redux-form';
+import { connect } from 'react-redux';
+import { Select, Input, Form, Button } from 'antd';
+import makeField from './helper';
+
+const InputSelect = makeField(Select);
+const TextArea = makeField(Input.TextArea);
+
+const { Option } = Select;
+
+const companyOption = company => (
+  <Option key={company.id} value={company.id}>
+    {company.name}
+  </Option>
+);
+
+const StudentSessionForm = ({ handleSubmit, disabled, companies }) => (
+  <Form onSubmit={handleSubmit}>
+    <Form.Item
```

Comment:
Form.Item should not be used here since it is also used in makeField

Reply:
Woopsie